community/kernel/src/main/java/org/neo4j/kernel/configuration/Config.java

```diff
@@ -587,6 +596,8 @@ public void updateDynamicSetting( String setting, String update )
 
         synchronized ( params )
         {
+            boolean oldDefault = false;
```

Comment:
it's really hard to guess why oldDefault and newDefault are called like this

Reply:
indeed, renaming